docs/topics/pages.rst

```diff
@@ -40,7 +40,7 @@ This example represents a typical blog post:
         body = RichTextField()
         date = models.DateField("Post date")
         feed_image = models.ForeignKey(
-            'wagtailimages.Image',
+            'media.Image',
```

Comment:
It might be better to encourage use of `wagtail.images.get_image_model_string()` here instead